Graphics_oM/Render/RenderText.cs

```diff
@@ -0,0 +1,65 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+
+using BH.oM.Base;
+using BH.oM.Geometry.CoordinateSystem;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.oM.Graphics
+{
+    [Description("Render text at a location and orientation in space with specified height, font and colour.")]
+    public class RenderText : IRender
+    {
+
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("The text to render.")]
+        public virtual string Text { get; set; } = "";
+
+        [Description("Cartesian to locate and orientate the text.")]
+
+        public virtual Cartesian Cartesian { get; set; } = new Cartesian();
+
+        [Description("Height of the text. Default is 1.")]
+
+        public virtual double Height { get; set; } = 1;
+
+        [Description("Font used to render the text. Default is Arial.")]
+
+        public virtual string FontName { get; set; } = "Arial";
+
+        [Description("Colour used to render the text. Default is Color.FromArgb(80, 255, 41, 105) BHoM pink.")]
```

Comment:
```suggestion
        [Description("Colour used to render the text. Default is BHoM pink (ARGB 80, 255, 41, 105).")]
```

Just because the users are less likely to care about the ARGB values and more what that colour is, I would put the most user-important information as the core bit :smile:

Reply:
@FraserGreenroyd it's not pink, it's Coral! You may be punished for such a mistake

Reply:
@alelom I was only going by what was there before, which shows the ARGB isn't the most important information cause I relied on the textual confirmation to know what it was 😝

Reply:
Whatever, my brain processes only 8 bit colours anyway, I can't tell Coral from orange or pink

Reply:
If it can't be rendered fully on a Sega Megadrive it's not worth rendering

Reply:
🦔💨